src/common/base64.ts

```diff
@@ -1,9 +1,17 @@
-export function urlDecode(a: string): string {
-  return atob(makeUrlUnsafe(a))
+export function decode(base64: string): string {
+  return Buffer.from(base64, 'base64').toString('binary')
```

Comment:
we inject `buffer` in the rollup config. in the browser it just uses Uint8Arrays under the hood. I'm using this pattern in `keystore-idb` and `webnative-filecoin` and quite like it